src/main/java/duke/Duke.java

```diff
@@ -0,0 +1,65 @@
+package duke;
+
+import helper.*;
+
+import helper.command.Command;
+
+import java.util.Scanner;
+
+/**
+ * Main class
+ */
+public class Duke {
+
+    private Storage storage;
+    private TaskList tasks;
+    private Ui ui;
+    private Parser parser;
+
+    /**
+     * Initializing duke
+     * @param filePath
```

Comment:
According to the coding standard, a line between description and parameters are required. Maybe an empty line here makes it more proper?
Edit: Same for every other javadoc.